### PR TITLE
Updating documentation section "Calculating Che resource requirements" with CPU and VS Code based workspaces details

### PR DESCRIPTION
**Pull request by themr0c**
Use Visual Studio Code in the example in Calculating Che resource requirements > Workspaces requirements

This pull requests contains also updates to clarify the tables, based on the observation of a recent deployment.

## What does this pull request change?

Firefox_Screenshot_2022-11-21T15-11-42 345Z

## What issues does this pull request fix or reference?

fixes: https://issues.redhat.com/browse/RHDEVDOCS-4562

## Specify the version of the product this pull request applies to

Che 7.56

## Pull Request checklist

The author and the reviewers validate the content of this pull request with the following checklist, in addition to the [automated tests](code_review_checklist.adoc).

- Any procedure:
  - [ ] Successfully tested.
- Any page or link rename:
  - [ ] The page contains a redirection for the previous URL.
  - Propagate the URL change in:
    - [ ] Dashboard [default branding data](https://github.com/eclipse-che/che-dashboard/blob/main/packages/dashboard-frontend/src/services/bootstrap/branding.constant.ts)
    - [ ] Chectl [constants.ts](https://github.com/che-incubator/chectl/blob/main/src/constants.ts)
- [ ] Builds on [Eclipse Che hosted by Red Hat](https://workspaces.openshift.com).
- [ ] the *`Validate language on files added or modified`* step reports no vale warnings.

### CODE REVIEW

**Review by azatsarynnyy — APPROVED** [review]

File: modules/administration-guide/pages/calculating-che-resource-requirements.adoc [hunk 1]
```diff
@@ -8,21 +8,166 @@
...
 = Calculating {prod-short} resource requirements
 
+The {prod-short} Operator, {devworkspace} Controller, and user workspaces consist of a set of pods.
+The pods contribute to the resource consumption in CPU and memory limits and requests.
+Learn how to calculate resources, such as memory and CPU, required to run {prod}.
 
-The {prod-short} Operator, {devworkspace} Controller, and user workspaces consist of a set of pods. The pods contribute to the resource consumption in terms of CPU and RAM limits and requests. Learn how to calculate resources, such as memory and CPU, required to run {prod}.
+.Procedure
+. Identify the workspace components explicitly specified in the `components` section of your devfile.
+When this section is empty {prod-short} only loads the implicit components.
```
Comment (deerskindoll) [inline modules/administration-guide/pages/calculating-che-resource-requirements.adoc]:
```suggestion
When this section is empty, {prod-short} only loads the implicit components.
```

File: modules/administration-guide/pages/calculating-che-resource-requirements.adoc [hunk 2]
```diff
@@ -8,21 +8,166 @@
...
+|256 Mi
+|64 Mi
+
+3+>s|Total
+>s|2.4 GiB
+>s|480 MiB
+|===
+
+
+. Sum up the resources required for each workspace, and multiply by the running workspaces count.
```
Comment (deerskindoll) [inline modules/administration-guide/pages/calculating-che-resource-requirements.adoc]:
```suggestion
. Sum up the resources required for each workspace, and multiply them by the running workspaces count.
```

File: modules/administration-guide/pages/calculating-che-resource-requirements.adoc [hunk 3]
```diff
@@ -8,21 +8,166 @@
...
+
+. Sum up with the server components requirements.
```
Comment (deerskindoll) [inline modules/administration-guide/pages/calculating-che-resource-requirements.adoc]:
```suggestion
. Sum up the server components requirements.
```

**Review by deerskindoll — COMMENTED** [review]
I made some minor suggestions, check them out.

**Review by deerskindoll — APPROVED** [review]
lgtm

File: modules/administration-guide/pages/calculating-che-resource-requirements.adoc [hunk 4]
```diff
@@ -8,21 +8,166 @@
...
+|
+|
+|
 
-include::partial$con_controller-requirements.adoc[leveloffset=+1]
+|Your editor
+|`workspace`
+|
+|
+|
```
Comment (themr0c) [inline modules/administration-guide/pages/calculating-che-resource-requirements.adoc]: Devfile never specifies editor

```suggestion
```

**Comment by deerskindoll** [comment]
Sync'd to Red Hat JIRA https://issues.redhat.com/browse/CRW-3621

**Pull request merged**